soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4745,9 +4737,90 @@ void DrawRandoEditor(bool& open) {
+                    "You got a %rBottle with Blue &Fire%w! Use it to melt Red Ice!",
+                    "You got a %rBottle with Blue &Fire%w! Use it to melt Red Ice!"),
+        GIMESSAGE(GI_BOTTLE_WITH_BIG_POE, ITEM_BIG_POE,
+                    "You got a %rBig Poe in a Bottle%w!&Sell it to the Ghost Shop!",
+                    "You got a %rBig Poe in a Bottle%w!&Sell it to the Ghost Shop!",
+                    "You got a %rBig Poe in a Bottle%w!&Sell it to the Ghost Shop!"),
+        GIMESSAGE(GI_BOTTLE_WITH_BLUE_POTION, ITEM_POTION_BLUE,
+                    "You got a %rBottle of Blue Potion%w!&Drink it to replenish your&%ghealth%w and %bmagic%w!",
+                    "You got a %rBottle of Blue Potion%w!&Drink it to replenish your&%ghealth%w and %bmagic%w!",
+                    "You got a %rBottle of Blue Potion%w!&Drink it to replenish your&%ghealth%w and %bmagic%w!"),
+        GIMESSAGE(GI_BOTTLE_WITH_FISH, ITEM_FISH,
+                    "You got a %rFish in a Bottle%w!&It looks fresh and delicious!&They say Jabu-Jabu loves them!",
+                    "You got a %rFish in a Bottle%w!&It looks fresh and delicious!&They say Jabu-Jabu loves them!",
+                    "You got a %rFish in a Bottle%w!&It looks fresh and delicious!&They say Jabu-Jabu loves them!"),
+        GIMESSAGE(GI_BOTTLE_WITH_BUGS, ITEM_BUG,
+                    "You got a %rBug in a Bottle%w!&They love to burrow in&dirt holes!",
+                    "You got a %rBug in a Bottle%w!&They love to burrow in&dirt holes!",
+                    "You got a %rBug in a Bottle%w!&They love to burrow in&dirt holes!"),
+        GIMESSAGE(GI_BOTTLE_WITH_FAIRY, ITEM_FAIRY, "You got a %rFairy in a Bottle%w!&Use it wisely!",
+                    "You got a %rFairy in a Bottle%w!&Use it wisely!",
+                    "You got a %rFairy in a Bottle%w!&Use it wisely!"),
+        GIMESSAGE(GI_BOTTLE_WITH_RED_POTION, ITEM_POTION_RED,
+                    "You got a %rBottle of Red Potion%w!&Drink it to replenish your&%ghealth%w!",
+                    "You got a %rBottle of Red Potion%w!&Drink it to replenish your&%ghealth%w!",
+                    "You got a %rBottle of Red Potion%w!&Drink it to replenish your&%ghealth%w!"),
+        GIMESSAGE(GI_BOTTLE_WITH_GREEN_POTION, ITEM_POTION_GREEN,
+                    "You got a %rBottle of Green Potion%w!&Drink it to replenish your&%bmagic%w!",
+                    "You got a %rBottle of Green Potion%w!&Drink it to replenish your&%bmagic%w!",
+                    "You got a %rBottle of Green Potion%w!&Drink it to replenish your&%bmagic%w!"),
+        GIMESSAGE(GI_BOTTLE_WITH_POE, ITEM_POE,
+                    "You got a %rPoe in a Bottle%w!&That creepy Ghost Shop might&be interested in this...",
+                    "You got a %rPoe in a Bottle%w!&That creepy Ghost Shop might&be interested in this...",
+                    "You got a %rPoe in a Bottle%w!&That creepy Ghost Shop might&be interested in this..."),
```

Comment:
doesn't need to happen now, but it might be nice to figure out a way to default to using the english string for all 3 when we don't have translations or something

ideally it'd be easy to go through the codebase and find messages that need to be translated, and i feel like having the english string repeated 3 times may not be the best way to make that happen

Reply:
Yeah because of how I did that I could pretty easily add a `#define` that just takes the one message and expands it to a struct that includes that one message 3 times. Call it something that indicates it's only in one language so it can show up in searches, like `GIMESSAGE_UNTRANSLATED` or something.

Reply:
> `GIMESSAGE_UNTRANSLATED`

this sounds perfect